direct/tpm2/pcr_test.go

```diff
@@ -0,0 +1,72 @@
+package tpm2
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-tpm/direct/structures/tpm"
+	"github.com/google/go-tpm/direct/structures/tpml"
+	"github.com/google/go-tpm/direct/transport/simulator"
+
+	"github.com/google/go-tpm/direct/structures/tpms"
+)
+
+func TestPCRReset(t *testing.T) {
+	thetpm, err := simulator.OpenSimulator()
+	if err != nil {
+		t.Fatalf("could not connect to TPM simulator: %v", err)
+	}
+	defer thetpm.Close()
+
+	PCRs, err := CreatePCRSelection([]int{16})
+	if err != nil {
+		t.Fatalf("Failed to create PCRSelection")
+	}
+
+	selection := tpml.PCRSelection{
+		PCRSelections: []tpms.PCRSelection{
+			{
+				Hash:      tpm.AlgSHA1,
+				PCRSelect: PCRs,
+			},
+		},
+	}
```

Comment:
If we don't use this struct anywhere else, could we inline it into the PCRRead struct below?

Reply:
Resolved.